eng/pipelines/templates/steps/workload-checkout.yml

```diff
@@ -1,13 +1,13 @@
 parameters:
-  sourceBranch: self
-  engBranch: eng
+  sourceBranchAlias: self
+  engBranchAlias: eng
```

Comment:
I updated these parameter names to be more accurate as it was confusing since they aren't actually the branch names, but aliases to the repository for those branches.